apps/acrs/behaviours/declaration.js

```diff
@@ -0,0 +1,48 @@
+const Utilities = require('../../../lib/utilities');
+
+const CheckUnder18InAggregatedValue = (aggregateArray, fieldLabel) => {
+  const aggregatedLength = aggregateArray.length;
+
+  let responseObject = '';
+  for (let i = 0; i < aggregatedLength; i++) {
+    responseObject = aggregateArray[i]
+      .fields
+      .map(v => v)
+      .filter(field => field.field === fieldLabel);
+
+    if (!Utilities.isOver18(responseObject[0].value)) {
```

Comment:
the `Utilities.isOver18` function checks that the user was over 18 at the time of evacuations (or that they were born before 23/08/2003.

May need to confirm with UCD here, but I think that this check should be that they are over 18 on today's date instead.